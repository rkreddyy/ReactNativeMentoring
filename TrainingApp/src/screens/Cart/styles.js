import { StyleSheet } from 'react-native';
import { BaseStyles } from '../../app.styles';

const styles = StyleSheet.create({
    wrapper: {
        backgroundColor: BaseStyles.colors.white,
        paddingLeft: BaseStyles.padding.m,
        paddingRight: BaseStyles.padding.m,
    },
    spinner: {
        marginTop: 'auto',
        marginBottom: 'auto',
    },
    productWrapper: {
        borderWidth: 1,
        borderColor: BaseStyles.colors.lightGrey,
        borderRadius: 5,
        marginTop: BaseStyles.margin.xs,
        padding: BaseStyles.padding.s,
        paddingLeft: BaseStyles.padding.l,
        paddingRight: BaseStyles.padding.l,
    },
    productContent: {
        flexDirection: 'row',
    },
    image: {
        width: 70,
        height: 70,
    },
    productInfo: {
        flex: 1,
    },
    name: {},
    price: {},
    productFooter: {
        flexDirection: 'row',
        justifyContent: 'space-between',
        alignItems: 'center',
        marginTop: BaseStyles.margin.m,
    },
    quantity: {
        width: 50,
        padding: BaseStyles.padding.s,
        borderWidth: 1,
        borderColor: BaseStyles.colors.lightBlue,
    },
    priceDetails: {
        borderWidth: 1,
        borderColor: BaseStyles.colors.lightGrey,
        borderRadius: 5,
        marginTop: BaseStyles.margin.xs,
        padding: BaseStyles.padding.s,
        paddingLeft: BaseStyles.padding.l,
        paddingRight: BaseStyles.padding.l,
    },
    priceDetailsTitle: {
        marginTop: BaseStyles.margin.s,
        marginBottom: BaseStyles.margin.s,
        fontWeight: 'bold',
        fontSize: BaseStyles.fontSize.m,
        textTransform: 'uppercase',
        color: BaseStyles.colors.grey,
    },
    totalCosts: {
        paddingTop: BaseStyles.padding.m,
        paddingBottom: BaseStyles.padding.m,
        flexDirection: 'row',
        justifyContent: 'space-between',
        borderTopWidth: 2,
        borderColor: BaseStyles.colors.lightGrey,
    },
    priceDetailsTotal: {
        paddingTop: BaseStyles.padding.m,
        flexDirection: 'row',
        justifyContent: 'space-between',
        borderTopWidth: 2,
        borderColor: BaseStyles.colors.lightGrey,
    },
    notesContainer: {
        marginTop: BaseStyles.margin.m,
        flexDirection: 'row',
        padding: BaseStyles.padding.xl,
        paddingRight: BaseStyles.padding.xxxl,
        paddingLeft: BaseStyles.padding.xxxl,
        backgroundColor: BaseStyles.colors.lightGrey,
    },
    notes: {
        marginLeft: BaseStyles.margin.m,
        color: BaseStyles.colors.grey,
        textTransform: 'capitalize',
    },
});

export default styles;
