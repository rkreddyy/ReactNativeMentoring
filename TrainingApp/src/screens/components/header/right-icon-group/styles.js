import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
    wrapper: {
        flexBasis: 70,
        justifyContent: 'space-between',
        flexDirection: 'row',
    },
});

export default styles;
