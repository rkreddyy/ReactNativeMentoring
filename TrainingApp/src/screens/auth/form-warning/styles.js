import { StyleSheet } from 'react-native';

import { BaseStylesSets, BaseStyles } from '../../../app.styles';

const styles = StyleSheet.create({
    errorContainer: {
        ...BaseStylesSets.containerRightLeftMargins,
        marginTop: 10,
        marginBottom: 10,
        backgroundColor: 'transparent',
    },
    error: {
        color: BaseStyles.colors.red,
        fontSize: BaseStyles.fontSize.s,
    },
});

export default styles;
