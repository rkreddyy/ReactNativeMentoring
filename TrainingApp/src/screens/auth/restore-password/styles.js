import { StyleSheet } from 'react-native';

import { BaseStyles, BaseStylesSets } from '../../../app.styles';

const styles = StyleSheet.create({
    container: {
        flex: 1,
    },
    backIcon: {
        width: BaseStyles.fontSize.xl,
        height: BaseStyles.fontSize.xl,
        marginLeft: BaseStyles.margin.l,
        marginTop: BaseStyles.margin.l,
    },
    title: {
        backgroundColor: 'transparent',
        fontSize: BaseStyles.fontSize.xl,
        textAlign: 'center',
        margin: BaseStyles.margin.l,
        color: BaseStyles.colors.lightBlue,
        marginTop: 200,
    },
    description: {
        ...BaseStylesSets.containerRightLeftMargins,
        color: BaseStyles.colors.black,
        textAlign: 'center',
        fontSize: BaseStyles.fontSize.s,
    },
    input: {
        ...BaseStylesSets.containerRightLeftMargins,
        alignSelf: 'stretch',
        backgroundColor: 'transparent',
        borderBottomWidth: 1,
        color: BaseStyles.colors.black,
        fontSize: BaseStyles.fontSize.m,
        marginTop: 100,
    },
    restoreText: {
        textAlign: 'center',
        color: BaseStyles.colors.white,
        textTransform: 'uppercase',
        fontSize: BaseStyles.fontSize.m,
    },
    restoreButton: {
        alignSelf: 'stretch',
        backgroundColor: BaseStyles.colors.blue,
        padding: BaseStyles.padding.m,
        margin: BaseStyles.margin.l,
    },
});

export const BackgroundGradientColors = ['#c4ddea', '#bdbad5', '#ebcdd6', '#f1dabe'];

export default styles;
