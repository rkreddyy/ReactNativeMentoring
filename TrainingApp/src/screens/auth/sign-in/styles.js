import { StyleSheet } from 'react-native';

import { BaseStyles, BaseStylesSets } from '../../../app.styles';

const styles = StyleSheet.create({
    container: {
        flex: 1,
    },
    title: {
        backgroundColor: 'transparent',
        fontSize: BaseStyles.fontSize.xl,
        textAlign: 'center',
        margin: BaseStyles.margin.l,
        color: BaseStyles.colors.lightBlue,
        marginTop: 260,
    },
    inputGroup: {
        flex: 1,
        alignItems: 'center',
    },
    input: {
        ...BaseStylesSets.containerRightLeftMargins,
        alignSelf: 'stretch',
        backgroundColor: 'transparent',
        borderBottomWidth: 1,
        color: BaseStyles.colors.black,
        fontSize: BaseStyles.fontSize.m,
        marginTop: 20,
    },
    restorePassword: {
        ...BaseStylesSets.containerRightLeftMargins,
        alignSelf: 'flex-end',
        marginTop: 20,
    },
    link: {
        ...BaseStylesSets.baseLink,
    },
    signUp: {
        ...BaseStylesSets.containerRightLeftMargins,
        alignSelf: 'center',
        marginBottom: BaseStyles.margin.l,
    },
    signInButton: {
        alignSelf: 'stretch',
        backgroundColor: BaseStyles.colors.blue,
        padding: BaseStyles.padding.m,
        margin: BaseStyles.margin.l,
    },
    signInText: {
        textAlign: 'center',
        color: BaseStyles.colors.white,
        textTransform: 'uppercase',
        fontSize: BaseStyles.fontSize.m,
    },
});

export const BackgroundGradientColors = ['#c4ddea', '#bdbad5', '#ebcdd6', '#f1dabe'];

export default styles;
