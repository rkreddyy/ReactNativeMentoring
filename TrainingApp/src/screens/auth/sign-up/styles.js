import { StyleSheet } from 'react-native';

import { BaseStyles, BaseStylesSets } from '../../../app.styles';

const styles = StyleSheet.create({
    container: {
        flex: 1,
    },
    title: {
        backgroundColor: 'transparent',
        fontSize: BaseStyles.fontSize.xl,
        textAlign: 'center',
        margin: BaseStyles.margin.l,
        color: BaseStyles.colors.lightBlue,
        marginTop: 100,
    },
    inputGroup: {
        flex: 1,
        alignItems: 'center',
    },
    input: {
        ...BaseStylesSets.containerRightLeftMargins,
        alignSelf: 'stretch',
        backgroundColor: 'transparent',
        borderBottomWidth: 1,
        color: BaseStyles.colors.black,
        fontSize: BaseStyles.fontSize.m,
        marginTop: 20,
    },
    restorePassword: {
        ...BaseStylesSets.containerRightLeftMargins,
        alignSelf: 'flex-end',
        marginTop: 20,
    },
    link: {
        ...BaseStylesSets.baseLink,
    },
    signIn: {
        ...BaseStylesSets.containerRightLeftMargins,
        alignSelf: 'center',
        marginBottom: BaseStyles.margin.l,
    },
    signInButton: {
        alignSelf: 'stretch',
        backgroundColor: BaseStyles.colors.blue,
        padding: BaseStyles.padding.m,
        margin: BaseStyles.margin.l,
    },
    signInText: {
        textAlign: 'center',
        color: BaseStyles.colors.white,
        textTransform: 'uppercase',
        fontSize: BaseStyles.fontSize.m,
    },
    backIcon: {
        width: BaseStyles.fontSize.xl,
        height: BaseStyles.fontSize.xl,
        marginLeft: BaseStyles.margin.l,
        marginTop: BaseStyles.margin.l,
    },
});

export const BackgroundGradientColors = ['#c4ddea', '#bdbad5', '#ebcdd6', '#f1dabe'];

export default styles;
