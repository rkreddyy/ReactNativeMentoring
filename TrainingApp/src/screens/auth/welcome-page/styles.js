import { StyleSheet } from 'react-native';
import { BaseStyles } from '../../../app.styles';

const styles = StyleSheet.create({
    container: {
        flex: 1,
    },
    title: {
        backgroundColor: 'transparent',
        fontSize: BaseStyles.fontSize.xl,
        textAlign: 'center',
        margin: BaseStyles.margin.l,
        color: BaseStyles.colors.lightBlue,
        marginTop: 220,
    },
});

export default styles;

export const BackgroundGradientColors = ['#c4ddea', '#bdbad5', '#ebcdd6', '#f1dabe'];
