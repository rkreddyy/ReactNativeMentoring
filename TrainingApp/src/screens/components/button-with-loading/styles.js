import { StyleSheet } from 'react-native';

import { BaseStyles } from '../../../app.styles';

const styles = StyleSheet.create({
    signInButton: {
        flex: 1,
        alignSelf: 'stretch',
        backgroundColor: BaseStyles.colors.blue,
        padding: BaseStyles.padding.m,
        margin: BaseStyles.margin.l,
        height: 40,
        alignItems: 'center',
        justifyContent: 'center',
    },
    signInButtonCollapsed: {
        width: 100,
        backgroundColor: BaseStyles.colors.blue,
        padding: BaseStyles.padding.m,
        margin: BaseStyles.margin.l,
        height: 40,
        alignItems: 'center',
        justifyContent: 'center',
        marginLeft: 'auto',
        marginRight: 'auto',
    },
    signInText: {
        color: BaseStyles.colors.white,
        textTransform: 'uppercase',
        fontSize: BaseStyles.fontSize.m,
    },
});

export default styles;
