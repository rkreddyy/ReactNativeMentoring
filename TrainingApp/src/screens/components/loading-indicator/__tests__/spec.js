import React from 'react';
import { shallow } from 'enzyme';

import { BaseStyles } from '../../../../app.styles';
import LoadingIndicator from '../loading-indicator';

describe(`LoadingIndicator component`, () => {
    const renderComponent = props => shallow(<LoadingIndicator {...props} />);
    describe('renders correctly with', () => {
        it('default props', () => {
            const component = renderComponent();

            expect(component).toMatchSnapshot();
        });

        it('with passed props', () => {
            const component = renderComponent({
                pointsCount: 4,
                color: BaseStyles.colors.white
            });

            expect(component).toMatchSnapshot();
        });
    });
});
