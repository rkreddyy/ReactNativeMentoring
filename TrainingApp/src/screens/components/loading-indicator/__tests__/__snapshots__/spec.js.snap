// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator component renders correctly with default props 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
      "minHeight": 40,
    }
  }
>
  <AnimatedComponent
    key="0"
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 7,
          "height": 7,
          "marginLeft": 2,
          "marginRight": 2,
          "width": 7,
        },
        Object {
          "backgroundColor": "#ffffff",
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        },
      ]
    }
  />
  <AnimatedComponent
    key="1"
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 7,
          "height": 7,
          "marginLeft": 2,
          "marginRight": 2,
          "width": 7,
        },
        Object {
          "backgroundColor": "#ffffff",
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        },
      ]
    }
  />
  <AnimatedComponent
    key="2"
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 7,
          "height": 7,
          "marginLeft": 2,
          "marginRight": 2,
          "width": 7,
        },
        Object {
          "backgroundColor": "#ffffff",
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        },
      ]
    }
  />
  <AnimatedComponent
    key="3"
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 7,
          "height": 7,
          "marginLeft": 2,
          "marginRight": 2,
          "width": 7,
        },
        Object {
          "backgroundColor": "#ffffff",
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        },
      ]
    }
  />
</View>
`;

exports[`LoadingIndicator component renders correctly with with passed props 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
      "minHeight": 40,
    }
  }
>
  <AnimatedComponent
    key="0"
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 7,
          "height": 7,
          "marginLeft": 2,
          "marginRight": 2,
          "width": 7,
        },
        Object {
          "backgroundColor": "#ffffff",
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        },
      ]
    }
  />
  <AnimatedComponent
    key="1"
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 7,
          "height": 7,
          "marginLeft": 2,
          "marginRight": 2,
          "width": 7,
        },
        Object {
          "backgroundColor": "#ffffff",
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        },
      ]
    }
  />
  <AnimatedComponent
    key="2"
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 7,
          "height": 7,
          "marginLeft": 2,
          "marginRight": 2,
          "width": 7,
        },
        Object {
          "backgroundColor": "#ffffff",
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        },
      ]
    }
  />
  <AnimatedComponent
    key="3"
    style={
      Array [
        Object {
          "backgroundColor": "#ffffff",
          "borderRadius": 7,
          "height": 7,
          "marginLeft": 2,
          "marginRight": 2,
          "width": 7,
        },
        Object {
          "backgroundColor": "#ffffff",
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        },
      ]
    }
  />
</View>
`;
